src/SWP/Bundle/FixturesBundle/DataFixtures/PHPCR/LoadHomepagesData.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Fixtures Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\FixturesBundle\DataFixtures\PHPCR;
+
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\Route;
+use SWP\Bundle\ContentBundle\Model\RouteInterface;
+use SWP\Bundle\FixturesBundle\AbstractFixture;
+use SWP\Bundle\WebRendererBundle\Doctrine\ODM\PHPCR\Tenant;
+use SWP\Component\MultiTenancy\Model\TenantInterface;
+
+class LoadHomepagesData extends AbstractFixture implements FixtureInterface, OrderedFixtureInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(ObjectManager $manager)
+    {
+        $tenants = [
```

Comment:
Quite a lot of things don't work with PHPCR, it seems. Makes me wonder whether it's a good idea to use it.